android/app/src/main/kotlin/coop/polypoly/polypod/FeatureCardAdapter.kt

```diff
@@ -60,81 +55,14 @@ class FeatureCardAdapter(
         view.setOnClickListener {
             // FIXME - navigation assumes we're coming from FirstFragment,
             // which might not necessary be true
-            authorize {
```

Comment:
This is one of the actual changes to the user flow: We used to call `authorize` before launching a feature, now it happens on startup.